docs/content/en/docs/user-guide/event-watcher.md

```diff
@@ -50,8 +50,10 @@ The full list of configurable `EventWatcher` fields are [here](/docs/user-guide/
 To register a new value corresponding to Event such as the above in the control-plane, you need to perform `pipectl`.
 And we highly recommend integrating a step for that into your CI workflow.
 
-According to [this guide](/docs/user-guide/command-line-tool/#installation), you first install it on your CI system or where you want to run.
-And then you grab the API key for the control-plane according to [this guide](/docs/user-guide/command-line-tool/#authentication).
+You first need to set-up the `pipectl`:
+
+- Install it on your CI system or where you want to run according to [this guide](/docs/user-guide/command-line-tool/#installation).
+- And then you grab the API key to which the `READ_WRITE` role is attached according to [this guide](/docs/user-guide/command-line-tool/#authentication).
```

Comment:
~And then you~

Reply:
@nakabonne I think you could remove the part `And then you` here, due to the bullet points pattern 😄

Reply:
Ahh, that's what you mean, thank you!